platform/security/filter/security-filter-csrf/pom.xml

```diff
@@ -70,17 +70,17 @@
                                         <limit implementation="org.codice.jacoco.LenientLimit">
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.91</minimum>
+                                            <minimum>0.92</minimum>
                                         </limit>
                                         <limit implementation="org.codice.jacoco.LenientLimit">
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.91</minimum>
+                                            <minimum>0.83</minimum>
```

Comment:
❓ These drops in coverage seem disproportionate to the code changed and number of unit tests added.

Reply:
I agree, all the previous test cases are still there and many more were added so I'm not sure why these coverage numbers dropped so much. I'll look into it a bit more.

Reply:
Made the filter logic a little cleaner and added a few more test scenarios, coverages look a lot better now